src/annotator/sidebar.tsx

```diff
@@ -70,7 +70,7 @@ export type SidebarContainerConfig = {
 /**
  * Create the iframe that will load the sidebar application.
  */
-function createSidebarIframe(config: SidebarConfig): HTMLIFrameElement {
+export function createSidebarIframe(config: SidebarConfig): HTMLIFrameElement {
```

Comment:
I'm exporting the function so that we can test the logic individually, but maybe it's worth moving it to some other module.

Reply:
Keeping it here but exporting it is fine.